### ISSUE TITLE
interested in building an ML agent (Not exactly an issue)

### ISSUE THREAD

**Issue by Mohit-Pala** [brubster/splashy-squid#1]
Hey, i found your game on itch.io.

I am interested in building a Reinforcement Learning agent for your game as a part of my capstone project.
I am still looking at various open source projects but really liked your game
Just wanted to reach out and see if you were cool with your game being controlled by Ai and whatnot

**Comment by brubster** [comment]
Yeah I'm cool with that! If your project can be open sourced (I understand if you are under school restrictions), would you mind linking it so I can take a look? :)

**Comment by Mohit-Pala** [comment]
Appreciate it, my capstone project starts next semester so ill update you then, as of right now i only have the idea and was looking for FOSS games on itch. I also plan to keep my project FOSS because i mainly want to contribute to the godot community.

**Comment by brubster** [comment]
Awesome. Good luck!
